.github/CONTRIBUTING.md

```diff
@@ -167,50 +167,56 @@ Additional references:
+1. An [Area Expert][area-expert] should also review the pull request.
+   - If the *author* does not meet the *reviewer*'s standards, the *reviewer* makes comments. A *maintainer* then removes the `Review - needed` label and adds 
+   the `Review - waiting on author` label. The *author* must address the comments and repeat from step 2.
+   - If the *author* meets the *reviewer*'s standards, the *reviewer* approves the PR. A maintainer then removes the `need review` label.
+1. Once the code review is completed, a *maintainer* merges the pull request after one business day to allow for additional critical feedback.
+
+#### Pull Request - Roles and Responsibilities
+
+1. The PR *author* is responsible for moving the PR forward to get it Approved.
+   This includes addressing feedback within a timely period and indicating feedback has been addressed by adding a comment and mentioning the specific *reviewers*.
+   When updating your pull request, please **create new commits** and **don't rewrite the commits history**. 
+   This way it's very easy for the reviewers to see diff between iterations.
+   If you rewrite the history in the pull request, review could be much slower.
+   Once the review is done, you can rewrite the history to make it prettier, if you like.
+   Otherwise it's likely would be squashed on merge to master by the *assignee*.
+1. *Reviewers* are any anyone who wants to contribute.
+   They are responsible for ensuring the code: addresses the issue being fixed, does not create new issues (functional, performance, or reliability), and implements proper design.
+   *Reviewers* should use the `Review changes` drop down to indicate they are done with their review.
+   - `Request changes` if you believe the PR merge should be blocked if your feedback is not addressed,
+   - `Approve` if you believe your feedback has been addressed or the code is fine as-is, it is customary (although not required) to leave a simple "Looks good to me" (or "LGTM") as the comment for approval.
+   - `Comment` if you are making suggestions that the *author* does not have to accept.
+   Early in the review, it is acceptable to provide feedback on coding style based on the published [Coding Guidelines](../docs/dev-process/coding-guidelines.md), however, after
+   the *author* has already pushed commits to address feedback, it is generally _not_ acceptable to focus on style issues as it disrupts the PR process.
+   Non-critical late feedback (after PR has been approved) can be submitted as a new issue or new pull request from the *reviewer*.
+1. *Assignee* who are always *Maintainers* ensure that proper review has occurred and if they believe one approval is not sufficient, the *maintainer* is responsible to add more reviewers.
+   An *assignee* may also be a reviewer, but the roles are distinct.
+   Once the PR has been approved and the CI system is passing, the *assignee* will merge the PR.
```

Comment:
> the assignee will merge the PR.

Should be `merge the PR after one business day`. We mentioned the `one business day` rule above in "Pull Request - Workflow"

Reply:
yes